inc/meta-box-callbacks.php

```diff
@@ -47,6 +47,7 @@ public function date( $post_id, $taxonomy, $multiples = false, $data = array(),
 		}
 		
 		if ( isset($data[$taxonomy] ) ) {
+			date_default_timezone_set('America/Chicago');
```

Comment:
I think I forgot to save a file before I committed...whoops. Thanks for catching it.